seed/xylem/marathon_sync.py

```diff
@@ -0,0 +1,53 @@
+from twisted.internet.defer import gatherResults
+from twisted.web.client import getPage
+
+from rhumba import RhumbaPlugin, cron
+
+
+class Plugin(RhumbaPlugin):
+    """
+    A plugin to periodically push an application group definition to Marathon.
+    """
+
+    getPage = getPage  # To stub out in tests.
+
+    def __init__(self, *args, **kw):
+        super(Plugin, self).__init__(*args, **kw)
+
+        self.marathon_host = self.config.get("marathon_host", "localhost")
+        self.marathon_port = self.config.get("marathon_port", "8080")
+        self.group_json_files = self.config["group_json_files"]
+
+    @cron(min="*/1")
+    def call_do_thing(self, args):
```

Comment:
Whoops. That started off as a "get this thing working" log method and then somehow turned into the real thing. I'll fix it.